### ISSUE TITLE
Set aria-label as static value

### ISSUE THREAD

**Issue by svinkle** [GoogleWebComponents/model-viewer#460]
### Description

Implementing an `aria-label` which doesn’t change its value may help with guaranteeing the label will be read correctly for a variety of assistive technology. Having the `aria-label` update dynamically, paired with the `aria-live` attribute may not have the best support. 

For example, JAWS paired with Edge browser does not announce the updated `aria-label` as expected. Chrome for Android announces the position of the stage, not what the model contains.

**Recommendation:** set the `aria-label` value as, “A 3D model of an astronaut” by default in order to guarantee this label announcement no matter where the user navigates from. Do not change this value throughout the experience.

```html
<canvas … aria-label="A 3D model of an astronaut">
  <!-- … -->
</canvas>
```

#### Live Demo
1. Go to [`<model-viewer>`](https://googlewebcomponents.github.io/model-viewer/index.html) with Chrome for Android
2. Load up [TalkBack screen reader](https://support.google.com/accessibility/android/answer/6007100?hl=en)
3. Swipe right with one finger until the `canvas` content is found

### Browser Affected

- [x] Chrome
- [x] Edge
- [ ] Firefox
- [ ] Helios
- [ ] IE
- [ ] Safari

### OS

- [x] Android
- [ ] iOS
- [ ] Linux
- [ ] MacOS
- [x] Windows

### Versions

- model-viewer: v0.1.1

**Comment by cdata** [comment]
@svinkle Thanks for this feedback. Are you proposing that live updates should no longer happen altogether, or just that we should find an alternative strategy to make them work?

**Comment by svinkle** [comment]
I'm proposing keeping the `aria-label` as static text, and adjusting the `aria-live` treatment as described in #461.

**Comment by cdata** [comment]
Ahh, yes, could you perhaps try again with the latest build on master? I discovered a bug related to what you are describing while working on #452 and [added a test](https://github.com/GoogleWebComponents/model-viewer/blob/master/src/test/features/controls-spec.js#L214-L221) for the preferred behavior.
